packages/trpc/server/routers/viewer/teams/inviteMember/utils.ts

```diff
@@ -96,6 +101,38 @@ export async function getUserToInviteOrThrowIfExists({
   return invitee;
 }
 
+export async function getUsersToInviteOrThrowIfExists({
+  usernameOrEmail,
+  teamId,
+  isOrg,
+}: {
+  usernameOrEmail: string[];
+  teamId: number;
+  isOrg?: boolean;
+}) {
+  // Check if user exists in ORG or exists all together
+
+  const orgWhere = isOrg && {
+    organizationId: teamId,
+  };
+  const invitees = await prisma.user.findMany({
+    where: {
+      OR: [{ username: { in: usernameOrEmail }, ...orgWhere }, { email: { in: usernameOrEmail } }],
+    },
+    include: { teams: { select: { teamId: true, userId: true, accepted: true }, where: { teamId: teamId } } },
```

Comment:
include memberships in order to check if the user can autojoin the team because he has an accepted membership to the org